app/scripts-browserify/markerclusterer.js

```diff
@@ -12,6 +12,11 @@
  * markers.
  */
 
+/**
+ * NOTICE: MarkerClusterer.prototype.calculator_ has been modified to support
```

Comment:
![](https://media.giphy.com/media/mMJygtkZIpu6Y/giphy.gif)

Potentially very dangerous.
Can we use something like [apply-patch](https://www.npmjs.com/package/apply-patch) instead?

Reply:
@tripox depends - how did markerclusterer get in there in the first place? If it was just added by hand i can't see we can do much besides mayby adding a patch-file and writing some other place that you have to remember to apply it?

Reply:
You're right. Let's just do it. 👍